### ISSUE TITLE
Doom Agh's buff getting overwrited by Rubick's doom.

### ISSUE THREAD

**Issue by NaGeKuroi** [ValveSoftware/Dota2-Gameplay#3169]
### Ability name

Doom

### Description

Rubick casting Doom on a Doom with the Agh's doom buff on itself will remove and overwrite the original doom but circle is still there.

### Example Match ID (and possibly Timestamp)

6760729339 27:36

**Comment by gatesyp** [comment]
Are you saying that when Doom casts his ultimate on himself, Rubick steals it and casts it on Doom again, the original ultimate is overwritten by Rubick's?

This sounds like it is working as designed

**Comment by ThinkPortals** [comment]
https://www.reddit.com/r/DotA2/comments/xhyw5e/bug_if_rubick_dooms_doom_after_doom_dooms_himself/

Doom aghs casts doom on himself, starts radiating doom on nearby enemies, all is well.
Then rubick steals it and uses doom on Doom. If Rubick doesn't have aghs, Doom becomes doomed. The visual AoE persists, and the buff tray shows a refreshed-duration green doom buff, but Doom is doomed.
If rubick has aghs, it's much the same, except it has the AoE doom you'd expect.
However, in both cases, Doom's original cast of doom on himself is overwritten, and no longer dooms enemies around him.
TL;DR, rubick doom overwrites Doom's aghs doom.

**Comment by NaGeKuroi** [comment]
> Are you saying that when Doom casts his ultimate on himself, Rubick steals it and casts it on Doom again, the original ultimate is overwritten by Rubick's?
> 
> This sounds like it is working as designed

Don't think that is how it should work but logically, this is one of the great way to counter doom aghs, you just have a pos 4/5 rubick with barely any item stealing the ult and confuse both doom and his allies.
By right ,it should be along the line of 1 doom buff on doom and 1 doom debuff on doom. Doom circle should still affect enemy and he himself also get silenced and muted with the debuff placed on it.

**Comment by Dota-Exp** [comment]
fixed in upcoming update
